internal/plugin.go

```diff
@@ -285,12 +284,11 @@ func (l *LuaPlugin) PreUse(version Version, previousVersion Version, scope UseSc
 		return "", err
 	}
 
-	function := l.pluginObj.RawGetString("PreUse")
-	if function.Type() == lua.LTNil {
+	if !l.HasFunction("PreUse") {
 		return "", nil
 	}
 
-	if err := l.vm.CallFunction(function, l.pluginObj, ctxTable); err != nil {
+	if err = l.CallFunction("PreUse", l.pluginObj, ctxTable); err != nil {
```

Comment:
looks like the second param `l.pluginObj` is useless, we can get this inside the `CallFunction`

Reply:
Ok, you are right. ;)